js/src/interface.js

```diff
@@ -283,12 +324,8 @@ let app = new Vue({
 
 // Fired when the external save button is clicked
 Fliplet.Widget.onSaveRequest(() => {
-
   // Send back the result
-  Fliplet.Widget.save(JSON.parse(JSON.stringify({
-    settings: settings,
-    result: app.result
-  }))).then(() => {
+  Fliplet.Widget.save(app.result).then(() => {
```

Comment:
any reason why you
- removed the JSON parse+stringify (which converts Vue reactive properties into a plain object)?
- you are only returning the results?